Online/State/AbstractCreatureState.cs

```diff
@@ -22,6 +22,7 @@ protected override RealizedPhysicalObjectState GetRealizedState(OnlinePhysicalOb
             if (onlineObject.apo.realizedObject is Player) return new RealizedPlayerState((OnlineCreature)onlineObject);
             if (onlineObject.apo.realizedObject is Overseer) return new RealizedOverseerState((OnlineCreature)onlineObject);
             if (onlineObject.apo.realizedObject is Creature) return new RealizedCreatureState((OnlineCreature)onlineObject);
+            if (onlineObject.apo.realizedObject is Fly) return new RealizedFlyState((OnlineCreature)onlineObject);
```

Comment:
Fly is creature so this never runs (the previous line returns)

Reply:
fixed